osquery/tables/system/windows/drivers.cpp

```diff
@@ -8,193 +8,237 @@
  *
  */
 
-#define WIN32_LEAN_AND_MEAN
-#include <Windows.h>
-#include <Winsvc.h>
-#include <psapi.h>
+#include <boost/regex.hpp>
```

Comment:
nit, do you mind having a double-newline between the categories of includes and also moving this boost include (a third-party dependency include) below the C++ include?